fossology/fossology
# feat(scanner): generate spdx report

## Description

Add capability to CI scanner to generate SPDX RDF, TAG VALUE, YAML and JSON reports. This features works only for repo scan.

### Changes

1. Modularize code.
2. Generate SPDX reports for repo scans.
3. Use better term allowlist instead of whitelist.

## How to test

Check the sample configuration and run the repo scan.

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2476"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
